### ISSUE TITLE
Support custom key bindings

### ISSUE THREAD

**Issue by ricelid** [jason-kane/PyYapf#38]
Please add support for custom key bindings.

**Comment by jason-kane** [comment]
Sure.  I'll need to figure out how, but it doesn't look particularly difficult.

**Comment by catch22** [comment]
You can just go to `Preferences -> Key Bindings` and add your favorite key bindings for the`yapf_selection` and `yapf_document` commands.

(The syntax is the same as in the three `.sublime-keymap` files in the repository.)

**Comment by ricelid** [comment]
Other packages have their key bindings next to their package settings, which is quite convenient.

**Comment by catch22** [comment]
You mean adding a menu entry? Good idea!

**Comment by ricelid** [comment]
@catch22 I tried putting this in my sublime-keymap file and it doesn't seem to work:
```
    {
    	"keys": ["super+option+a"], "command": "yapf_document",
    	"context":  [{"key": "selector", "operator": "equal", "operand": "source.py"}]
    }
```
Maybe user error?

**Comment by jason-kane** [comment]
Pretty trival to add:
https://github.com/SublimeGit/SublimeGit/blob/master/Main.sublime-menu#L28  

but it looks like that won't solve your problem, just make it easier to open the keymap file.

**Comment by jason-kane** [comment]
http://docs.sublimetext.info/en/latest/reference/key_bindings.html is a bit internally inconsistent.  "option" isn't listed as a valid modifier but it is used in an example -- of what not to do.

**Comment by ricelid** [comment]
Replaced "option" wtih "alt", still doesn't seem to work.
Does something similar work for you?

**Comment by catch22** [comment]
Did you add `[...]` around it?

**Comment by catch22** [comment]
Try copying and adapting our `.sublime-keymap` file (assuming it works...).

**Comment by ricelid** [comment]
Figured it out. It needed to be `source.python`, not `source.py`.
Here's what works:
```json
    {
    	"keys": ["super+lalt+a"], "command": "yapf_selection",
    	"context":  [{"key": "selector", "operator": "equal", "operand": "source.python"}]
    }
```
I think we should keep this issue open because AFAIK there's currently no way to remove the default key bindings, just adding new ones. Also the convenience of having the package's key bindings show up in the normal spot in the menu would be nice.

**Comment by catch22** [comment]
I don't think you need the `context` part -- the `yapf_selection` (etc.) command only runs when the view contains Python code.

**Comment by catch22** [comment]
I added menu entries for the README & the Key Bindings (and also improved the Settings one). Looks good?

**Comment by jason-kane** [comment]
looks good to me

**Comment by jason-kane** [comment]
Released at v0.13.0, nicely done.
